client/globals.d.ts

```diff
@@ -175,6 +175,24 @@ declare global {
 		timeFormat: string;
 	};
 
+	const woopaymentsPaymentMethodDefinitions: Record<
+		string,
+		{
+			id: string;
+			stripe_key: string;
+			title: string;
+			description: string;
+			capabilities: string[];
+			currencies: string[];
+			countries: string[];
+			allows_manual_capture: boolean;
+			allows_pay_later: boolean;
+			accepts_only_domestic_payment: boolean;
+			settings_icon_url: string;
+			icons: PaymentMethodIcons;
```

Comment:
I don't see `capabilities`, `icons`, `countries` being passed from the backend in `includes/payment-methods/Configs/Utils/PaymentMethodUtils.php`. Should we remove them?
```suggestion
			id: string;
			stripe_key: string;
			title: string;
			description: string;
			currencies: string[];
			allows_manual_capture: boolean;
			allows_pay_later: boolean;
			accepts_only_domestic_payment: boolean;
			settings_icon_url: string;
```

Reply:
Ahhh...sorry about that. Copied/pasted the outdated definition. Updated in 23be5cbee